docs/userguide.md

```diff
@@ -62,8 +62,7 @@ is a detailed list of binaries required.
 
 The federation deployment depends on `kubebuilder`, `etcd`, `kubectl`, and
 `kube-apiserver` >= v1.13 being installed in the path. The `kubebuilder`
-([v1.0.8](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1.0.8)
-as of this writing) release packages all of these dependencies together.
+([v1.0.8](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1.0.8)) release packages all of these dependencies together.
```

Comment:
Why is this change suggested?

Reply:
@marun I was under the impression that they have released v1.0.9 rc for `Kubebuilder` but it looks like that release was meant for the `controller-tools` project (sub-project for `Kubebuilder`). Let me revert this change.